flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/BufferBlocker.java

```diff
@@ -42,15 +42,15 @@
 	 * Starts a new sequence of buffers and event without reusing the same resources and
 	 * returns the current sequence of buffers for reading.
 	 *
-	 * @return The readable sequence of buffers and events, or 'null', if nothing was added.
+	 * @return The readable sequence of buffers or events, or 'null', if nothing was added.
 	 */
 	BufferOrEventSequence rollOverWithoutReusingResources() throws IOException;
 
 	/**
 	 * Starts a new sequence of buffers and event reusing the same resources and
 	 * returns the current sequence of buffers for reading.
 	 *
-	 * @return The readable sequence of buffers and events, or 'null', if nothing was added.
+	 * @return The readable sequence of buffers or events, or 'null', if nothing was added.
```

Comment:
This doesn't improve the doc. The sequence can contain both buffers **AND** events, not just one type of them.